Source/RSCodeGenerator.swift

```diff
@@ -104,9 +104,13 @@ open class RSAbstractCodeGenerator : RSCodeGenerator {
         // Top spacing          = 1.5
         // Bottom spacing       = 2
         // Left & right spacing = 2
-        // Height               = 28
         let width = length + 4
-        let size = CGSize(width: CGFloat(width), height: 28)
+        // Calculate the correct aspect ratio, so that the resulting image can be resized to the target size
+        var height = width
```

Comment:
`height` is better to be a fixed value in case the width is too big, so I prefer not to update the const value of height.

1. Revert line 107
2. var height = 28

Reply:
I see your point, however, this functions returns a static image and not vector graphics. It Is quite difficult to scale the image afterwards because of the "magic number" 28.
However, your solution is good. For backwards compatibility, the "var height = 28" is a good idea and then only if the targetSize Is defined, the code will use the scaling friendly aspect ratio calculation.